cpp/include/cudf/io/types.hpp

```diff
@@ -489,6 +489,20 @@ class column_in_metadata {
     return *this;
   }
 
+  /**
+   * @brief Specifies whether this column should be written as binary or string data
+   * Only valid for the following column types:
+   * string
```

Comment:
```suggestion
   * Only valid for the `string` column type.
```
Do we foresee the conversion to binary being applicable to multiple data types?
(I think the likes of `VARCHAR`, etc. might be handled by `STRING`.)

Reply:
list<int8> is the only one that comes to mind and is the follow-up PR

Reply:
Ah, I stand corrected.